### DISCUSSION TITLE
How to define the action column?

### DISCUSSION THREAD

**Discussion by ArtMin96** [rappasoft/laravel-livewire-tables#714] [Q&A] [answered]
How to define the action column?

**SQLSTATE[42S22]: Column not found: 1054 Unknown column 'entity.actions' in 'field list'**

```php
Column::make('Actions')
    ->excludeFromColumnSelect()
    ->format(
        fn($value, $row, Column $column) => view('tables.cells.call')->withValue($value)
    )
```

**Comment by demeus** [comment, marked as answer]
```
 Column::make("Actions")
                ->label(
                    fn($row, Column $column) => view('tables.cells.call')->withValue($row)
                ),
```

**Reply by ArtMin96** [reply]
Thanks

**Reply by itsmeduard** [reply]
> ```
>  Column::make("Actions")
>                 ->label(
>                     fn($row, Column $column) => view('tables.cells.call')->withValue($row)
>                 ),
> ```
Hi! You save my life! Thank you so much.

**Reply by faridsa** [reply]
In my case it fires and SQL error
`SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.actions' in 'field list'` because there is not an Actions column in the table.

**Comment by Nemachtilli** [comment]
`Column::make("Actions")
                ->label(
                    fn($row, Column $column) => view('users.admin.users.datatables.actions')->with(['user' =>$row])
                ),`

**Comment by Nemachtilli** [comment]
`Column::make('Action')
            ->label(function ($row, Column $column) {
                    return view('path.view', ['user' => $row]);
                },
            ),`
